### ISSUE TITLE
Management Engine status menu

### ISSUE THREAD

**Issue by miczyg1** [Dasharo/dasharo-issues#140]
**The problem you're addressing (if any)**
Lack of management Engine status in the firmware. Management Engine may be often disabled or non functional for various reasons during POST. The firmware will disable HECI device and save the last known state in SMBIOS for proprietary firmware. Dasharo lacks this mechanism. 

**Describe the solution you'd like**
The Management Engine status menu would show either the current status retrieved from HECI or the SMBIOS if HECI is hidden or not functional. Implement SMBIOS ME status if HECI is not functional in coreboot. Also implement showing ME state when HAP bit is enabled or ME is soft disabled.

**Where is the value to a user, and who might that user be?**
Better UX and proof that ME is disabled. May also show some Boot Guard status.

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by pietrushnic** [comment]
@miczyg1 isn't it duplicate of #111 ?

**Comment by miczyg1** [comment]
@pietrushnic not exactly. #111 aims to implement options to trigger ME disable or set the HAP bit. This request aims to present in human readable form the state of ME based on hardware registers. Although these two requests may be closely related, i.e. land in the same submenu.

**Comment by miczyg1** [comment]
@pietrushnic yes it can be viewed like that. But note aggregation of features doesn't play well for us. it is easier to deliver smaller blocks that still have value for users.
